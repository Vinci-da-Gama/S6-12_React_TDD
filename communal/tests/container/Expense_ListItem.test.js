import React from 'react';
import { shallow } from 'enzyme';

import ExpenseListItem from '../../src/components/livingExpense/Expense_ListItem';
import fixtureExpenses from '../fixtures/expenses.fixture';

test('should render dashboard correctly.', () => {
    const wrapper = shallow(<ExpenseListItem {...fixtureExpenses[0]} />);
    expect(wrapper).toMatchSnapshot();
});
