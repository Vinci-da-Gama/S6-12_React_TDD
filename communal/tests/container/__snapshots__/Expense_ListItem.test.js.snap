// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly. 1`] = `
<div>
  <ListGroup
    tag="ul"
  >
    <ListGroupItem
      className="justify-content-between"
      tag="li"
    >
      <Link
        replace={false}
        to="/edit/1"
      >
        <h3>
          Gum
        </h3>
      </Link>
      <i>
        Date: 
      </i>
      January 1st, 1970
      <Badge
        color="secondary"
        pill={true}
        tag="span"
      >
        <Label
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Amount: 
        </Label>
        $1.95
      </Badge>
      <i>
            Note: 
      </i>
      <span />
    </ListGroupItem>
  </ListGroup>
</div>
`;
